aiobotocore/args.py

```diff
@@ -34,6 +34,7 @@ def get_client_args(self, service_model, region_name, is_secure,
 
         signing_region = endpoint_config['signing_region']
         endpoint_region_name = endpoint_config['region_name']
+
         if signing_region is None and endpoint_region_name is None:
```

Comment:
from what I see diffing aiobotocore against 1.15.3 this `if signing_region` was removed along with `if isinstance(client_config, AioConfig):` and adding `connector_args` to `create_endpoint` call.  I think when you did you diff you did aiobotocore <-> 1.13.14 instead of against 1.15.3.  If you want to allow me to change your branch I can help fix, otherwise mind doing another diff against all the files?

Reply:
actually looked at the diff and I think this is the only file that needs to be updated

Reply:
You sure we want to remove
```
        if isinstance(client_config, AioConfig):
            connector_args = client_config.connector_args
        else:
            connector_args = None
```
?

Reply:
As i believe, if you pass config in via AioSession.create_client, get_default_client_config should return None, config is not altered so could end up not being an AioConfig type

Reply:
whoops, hah, ya that was our change, my bad

Reply:
probably later we should comment our non obvious await changes